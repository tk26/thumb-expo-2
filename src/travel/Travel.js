import React, { Component } from 'react';
import { View, Text, Button, Image } from 'react-native';

export default class Travel extends Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <View>
                <Image
                    source={require('./../../assets/thumb-horizontal-logo.png')}
                />
                
                <View>
                    <Text>
                        What do you want to do?
                    </Text>
                </View>

                <Button title="Ride" style={{ alignSelf: 'center' }}
                    onPress={() => this.props.navigation.navigate('RideStep1')} />

                <Button title="Drive" style={{ alignSelf: 'center' }}
                    onPress={() => this.props.navigation.navigate('DriveStep1')} />
            </View>
        );
    }
}