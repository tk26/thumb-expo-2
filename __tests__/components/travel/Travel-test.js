import React from 'react';
import Travel from '../../../src/components/travel/Travel';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<Travel />).toJSON();
  expect(tree).toMatchSnapshot();
});
