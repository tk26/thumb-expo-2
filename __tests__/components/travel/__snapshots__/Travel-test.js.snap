// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Image
    source={1}
  />
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      What do you want to do?
    </Text>
  </View>
  <View
    accessibilityComponentType="button"
    accessibilityLabel={undefined}
    accessibilityTraits={
      Array [
        "button",
      ]
    }
    accessible={true}
    collapsable={undefined}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={undefined}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Ride
      </Text>
    </View>
  </View>
  <View
    accessibilityComponentType="button"
    accessibilityLabel={undefined}
    accessibilityTraits={
      Array [
        "button",
      ]
    }
    accessible={true}
    collapsable={undefined}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={undefined}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Drive
      </Text>
    </View>
  </View>
</View>
`;
